### DISCUSSION TITLE
為什麼github儲存庫與本機儲存庫無法同步

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#245] [Q&A] [answered]
我在github的儲存庫做了更動，README.md檔案現狀如下
![image](https://user-images.githubusercontent.com/86505454/154251319-8a62522b-e00b-4145-b828-bba5958f44d5.png)
但我使用visual studio code開啟使用git clone將github儲存庫放到我桌面上，而產生的儲存庫，裡面的README.md檔案居然長成下面這樣。
![image](https://user-images.githubusercontent.com/86505454/154251689-89e0b1aa-84b4-4469-9c28-eaadd420a3b9.png)
怎麼一點都沒變？沒變也就罷了，為什麼visual studio code中的git沒有動作？情況如下所示。
<img width="960" alt="git怎麼沒有動作" src="https://user-images.githubusercontent.com/86505454/154253117-085c5eb9-1bdc-4305-a4e9-5095297dfbbc.png">

**Comment by fatsoap** [comment, marked as answer]
左下角有個重新整理的符號，那個按鈕可以同步遠端和本地端，每次有新的更新要上傳或是下載遠端的更新都要按一下

**Reply by enoch-lightning** [reply]
同學你真的很厲害，應該是資工系的大大吧？

**Reply by enoch-lightning** [reply]
我把這個討論(discussion)的內容總結在如下的兩部影片中。
[vs code連動github markdown預覽](https://youtu.be/_aryhCFODHg)
[vs code連動github markdown預覽 補充](https://youtu.be/Ufk-PSHVuyI)

**Reply by fatsoap** [reply]
沒，剛好會而已，我也是菜雞XD

**Reply by enoch-lightning** [reply]
你都會寫網頁和聊天機器人了，我都不會。
😆
